packages/cli/init/src/utils/docsUrlUtils.ts

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import { load } from "cheerio";
+
+export type DocsUrl = ReadMeDocsUrl;
+
+export interface DocsUrlInterface {
+    getAllOpenApiUrls(): Promise<string[]>;
+    getBaseUrl(): string;
+    isUrlValid(): boolean;
+}
+
+class DocsUrlBase implements DocsUrlInterface {
```

Comment:
what do you think of changing the naming to: 

```
class DocsInstance implements DocsInterface {

}

class ReadmeInstance extends DocsInstance {

}
```

Reply:
what about 
`class DocsImpl implements DocsInterface {

}

class ReadmeImpl extends DocsInstance {

}`? Noticed [impl](https://sourcegraph.com/github.com/fern-api/fern@add-cli-option/-/blob/packages/cli/cli/src/cli-context/TaskContextImpl.ts?L31) is used in the codebase in other places: https://sourcegraph.com/github.com/fern-api/fern@add-cli-option/-/blob/packages/cli/cli/src/cli-context/TaskContextImpl.ts?L31